doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +47,140 @@ length, :math:`L_0`,
+
+Implementation Discussion and Example of Radial and Axial Thermal Expansion
+===========================================================================
+
+This section provides an example thermal expansion calculation for a simple cylindrical component in the following figure:
+
+.. figure:: /.static/axial_expansion_simple.png
+
+    Illustration of Simple Cylinder Radial and Axial Thermal Expansion
+
+from a reference temperature of 20°C to 1000°C with example material properties and dimensions as shown in this table:
+
+.. list-table:: Example Component Properties for Thermal Expansion
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Property
+     - Example
+   * - Material
+     - Steel
+   * - Radius
+     - 0.25 cm
+   * - Height
+     - 5.0 cm
+   * - Reference Temperature
+     - 20°C
+   * - Density
+     - 1.0 g/cc
+   * - Mean Coefficient Thermal Expansion
+     - 2 x 10-6 1/°C
+
+This process is performed iteratively for each component as temperatures change. One important note about the thermal expansion implementation is that components are constrained within blocks, so the height of the component is determined by the height of its parent Block. What follows is a simple example illustrating the behavior of thermal expansion for a Block containing a single Component.
+
+.. list-table:: Example Calculation of Radial and Axial Thermal Expansion for a Cylindrical Component
+   :widths: 33 33 33
+   :header-rows: 1
+
+   * - Component Temperature
+     - 20°C
+     - 1000°C
+   * - Radius
+     - 0.25 cm
+     - 0.251 cm
+   * - Height
+     - 5.0 cm
+     - 5.01 cm
+   * - Volume
+     - 0.982 :math:`cm^3`
+     -  0.988 :math:`cm^3`
+   * - Density
+     - 1.0 g/cc
+     - 0.994 g/cc
+   * - Mass
+     - 0.982 g
+     - 0.982 g
+
+.. math::
+    \vec{r} = 0.25 cm \cdot (1 + (2 x 10^-6 \frac{1}{°C})(1000°C − 20°C)) = 0.251 cm
+
+.. math::
+    \vec{r} = 5.0 cm \cdot (1 + (2 x 10^-6 \frac{1}{°C})(1000°C − 20°C)) = 5.01 cm
+
+.. math::
+    \vec{V} = \pi \cdot (0.251 cm)^2 \cdot 5.01 cm = 0.988 cm^3
+
+.. math::
+    \vec{\rho} = \frac{1.0 \frac{g}{cc}}{(1 + (2 x 10^-6 \frac{1}{°C})(1000°C − 20°C))} = 0.994 \frac{g}{cc}
+
+.. math::
+    \vec{m} = 0.994 \frac{g}{cc} \cdot 0.988 cm^3 = 0.982 g
+
+When two or more components exist within the Block, the overall height change of the Block is driven by a target Component (e.g., fuel). The target is selected to ensure some Parameter of interest (e.g., linear heat generation rate) is evaluate correctly. In this case, the height of the Block containing the multiple components is only expanded to meet the axial expansion of the target Component and the remaining solid components have their densities further adjusted to preserve mass within the Assembly.
```

Comment:
we should try and get consistency between Component / component and Block / block.